spec/models/course_spec.rb

```diff
@@ -25,8 +25,8 @@
     let!(:course1) do
       FactoryGirl.create(:course,
         mooc_provider_id: provider.id,
-        start_date: Time.zone.local(2015, 03, 15),
-        end_date: Time.zone.local(2015, 03, 17),
+        start_date: Time.zone.local(2015, 0o3, 15),
```

Comment:
Its an octal representation (as it was before as well) but more explicit. I'll refactor that to remove the leading `0o`.